src/parse/asp/grammar_parse.go

```diff
@@ -430,41 +428,47 @@ func (p *parser) parseUnconditionalExpressionInPlace(e *Expression) {
 	}
 }
 
-func concatStrings(lhs *ValueExpression, rhs *ValueExpression) *ValueExpression {
-	// If they're both fStrngs
+func concatStrings(lhs *ValueExpression, rhs *ValueExpression) {
```

Comment:
This is doing the same thing just isn't returning a value to be clearer that it's mutating the parameter?